spec/integration/build-swift/after/Example/Example.xcodeproj.yaml

```diff
@@ -0,0 +1,168 @@
+
+File References:
```

Comment:
The way these integration tests work is that xcres is applied with specific arguments to an Example Project.
During this process, xcres does its changes to the source project.
The generated project's YAML representation is then diffed against a reference YAML file to confirm, that xcres does what expected.